projects/packages/jetpack-mu-wpcom/src/features/wpcom-global-styles/index.php

```diff
@@ -789,7 +792,7 @@ function wpcom_site_has_global_styles_in_personal_plan( $blog_id = 0 ) {
 		$blog_id = get_current_blog_id();
 	}
 
-	$cache_key                          = "global-styles-on-personal-02-2025-$blog_id";
+	$cache_key                          = "global-styles-on-personal-03-2025-$blog_id";
```

Comment:
I changed it now in case we forget to change it later. If we keep the same cache key users won't get assigned to the new experiment variation once we set up the experiment.